packages/lesswrong/components/tagging/TagPage.tsx

```diff
@@ -446,17 +447,35 @@ const TagPage = ({classes}: {
             tag={tag}
           />}
           {tag.sequence && <TagIntroSequence tag={tag} />}
-          {!tag.wikiOnly && <AnalyticsContext pageSectionContext="tagsSection">
-            <PostsListHeading tag={tag} query={query} classes={classes} />
-            <PostsList2
-              terms={terms}
-              enableTotal
-              tagId={tag._id}
-              itemsPerPage={200}
-            >
-              <AddPostsToTag tag={tag} />
-            </PostsList2>
-          </AnalyticsContext>}
+          {!tag.wikiOnly && <>
+            <AnalyticsContext pageSectionContext="tagsSection">
+              <PostsListHeading tag={tag} query={query} classes={classes} />
+              <PostsList2
+                terms={terms}
+                enableTotal
+                tagId={tag._id}
+                itemsPerPage={200}
+              >
+                <AddPostsToTag tag={tag} />
+              </PostsList2>
+            </AnalyticsContext>
+            <DeferRender ssr={false}>
```

Comment:
I don't feel strongly about adding this, seemed reasonable since this section is at the bottom and will rarely have content, but let me know if you disagree